### PR TITLE
Location test

### PR DESCRIPTION
**Pull request by lydiazheng**
there are still one failure and three errors.

 1) Failure:
UsersEditTest#test_successful_edit_with_friendly_forwarding [/Users/lynn/Desktop/GroupProject_cmpt276/test/integration/users_edit_test.rb:30]:
Expected response to be a redirect to <http://www.example.com/users/830138774/edit> but was a redirect to <http://www.example.com/login>.
Expected "http://www.example.com/users/830138774/edit" to be === "http://www.example.com/login".

  2) Error:
UsersEditTest#test_unsuccessful_edit:
NoMethodError: undefined method `session' for nil:NilClass
    test/integration/users_edit_test.rb:11:in `log_in_as'
    test/integration/users_edit_test.rb:15:in `block in <class:UsersEditTest>'

  3) Error:
UserMailerTest#test_account_activation:
StandardError: No fixture named 'michael' found for fixture set 'users'
    test/mailers/user_mailer_test.rb:6:in `block in <class:UserMailerTest>'

  4) Error:
UserMailerTest#test_password_reset:
StandardError: No fixture named 'michael' found for fixture set 'users'
    test/mailers/user_mailer_test.rb:18:in `block in <class:UserMailerTest>'

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -45,8 +45,6 @@
     t.datetime "updated_at",     null: false
   end
 
-  add_index "games", ["organizer_id"], name: "index_games_on_organizer_id", using: :btree
```
Comment (ghost) [inline db/schema.rb:48]: Lydia, please undo the changes to the schema.rb file, the schema should not be changed.

**Comment by lydiazheng** [comment]
1) Failure:
UsersEditTest#test_successful_edit_with_friendly_forwarding [/Users/lynn/Desktop/GroupProject_cmpt276/test/integration/users_edit_test.rb:30]:
Expected response to be a redirect to http://www.example.com/users/830138774/edit but was a redirect to http://www.example.com/login.
Expected "http://www.example.com/users/830138774/edit" to be === "http://www.example.com/login".

  2) Error:
UsersEditTest#test_unsuccessful_edit:
NoMethodError: undefined method `session' for nil:NilClass
    test/integration/users_edit_test.rb:11:in`log_in_as'
    test/integration/users_edit_test.rb:15:in `block in class:UsersEditTest'
